src/Loop/Driver.php

```diff
@@ -40,7 +40,7 @@ public function defer(callable $callback, $data = null);
     /**
      * Delay the execution of a callback.
      *
-     * The delay is a minimum and approximate, accuracy is not guaranteed.
+     * The delay is a minimum and approximate, accuracy is not guaranteed. Order of calls MUST be determined by which timers expire first.
```

Comment:
That was implied here (one delay always expires first as it is declared a few microseconds earlier...)

How would you word it?